vertical-pod-autoscaler/pkg/admission-controller/resource/pod/patch/resource_updates_test.go

```diff
@@ -249,6 +270,26 @@ func TestClalculatePatches_ResourceUpdates(t *testing.T) {
 				addAnnotationRequest([][]string{{cpu}}, limit),
 			},
 		},
+		{
+			name: "replacement cpu limit from container status",
+			pod: test.Pod().
+				AddContainer(core.Container{}).
```

Comment:
Ditto here, wonder if we should set some request values to more accurately reflect a real scenario?